overlays/jaeger/otel-collector.Deployment.yaml

```diff
@@ -13,3 +13,5 @@ spec:
           env:
             - name: JAEGER_HOST
               value: jaeger-collector
+            - name: JAEGER_OTLP_GRPC_PORT
+              value: 4320
```

Comment:
Are we missing `- 'JAEGER_OTLP_HTTP_PORT=4321'`?

Reply:
We absolutely are, great catch!